CHANGELOG.md

```diff
@@ -160,59 +160,15 @@ Workaround -> When using onchain permissioning, ensure bootnodes are also valida
 ### Additions and Improvements 
```

Comment:
This file may need a hand merge.  I don't think there should be any diffs.

Reply:
Hand merged.